app/javascript/components/rooms/CantCreateRoom.jsx

```diff
@@ -1,28 +1,44 @@
-import React from 'react';
+import React, { useState } from 'react';
 import { Card, Stack } from 'react-bootstrap';
 import Button from 'react-bootstrap/Button';
 import { useTranslation } from 'react-i18next';
-import Logo from '../shared_components/Logo';
+import { useNavigate } from 'react-router-dom';
 
-// This page is shown is the current_user does NOT have CreateRoom permission
+// This page is shown if the user does NOT have the CreateRoom permission
 export default function CantCreateRoom() {
   const { t } = useTranslation();
+  const navigate = useNavigate();
+
+  const [meetingUrl, setMeetingUrl] = useState('');
+
+  const handleChange = (event) => {
+    setMeetingUrl(event.target.value);
+  };
+
+  // Parse the 12 keys friendly id from the input
+  const regex = /(\w{3}-\w{3}-\w{3}-\w{3})/;
+  const parsedUrl = regex.exec(meetingUrl);
```

Comment:
Based on the little I know, you should be using `meetingUrl.match(regex)`. `.exec` was meant to be used in an array or loop or something like that

Reply:
They seem to be the same. match() syntax is cleaner IMO